src/background.js

```diff
@@ -427,6 +426,7 @@ async function runWallet() {
     walletProcess.stdout.on('data', async function(data) {
       console.info('stdout: ' + data.toString())
       status = STATUS.READY
+      win.webContents.send('running')
```

Comment:
Why are you sending this message right above loaded? Doing this you cannot differentiate the two steps.